apps/web/src/store/index.ts

```diff
@@ -115,18 +119,28 @@ export const _hydrationReducer: typeof rootReducer = (state, action) => {
   return rootReducer(state, action) as RootState
 }
 
-export const makeStore = (initialState?: Partial<RootState>): EnhancedStore<RootState, Action> => {
+type MakeStoreOptions = {
+  skipBroadcast?: boolean
+}
+export const makeStore = (
+  initialState?: Partial<RootState>,
+  options?: MakeStoreOptions,
+): EnhancedStore<RootState, Action> => {
+  setBaseUrl(GATEWAY_URL)
+
   const store = configureStore({
     reducer: _hydrationReducer,
     middleware: (getDefaultMiddleware) => {
       listeners.forEach((listener) => listener(listenerMiddlewareInstance))
-      return getDefaultMiddleware({ serializableCheck: false }).concat(middleware)
+      return getDefaultMiddleware({ serializableCheck: false }).concat(cgwClient.middleware).concat(middleware)
     },
     devTools: !IS_PRODUCTION,
     preloadedState: initialState,
   })
 
-  listenToBroadcast(store)
+  if (!options?.skipBroadcast) {
+    listenToBroadcast(store)
+  }
```

Comment:
in jsdom the BroadcastChannel object was actually missing. Note that it is generally available in the Browser and in Node. With the jest-fixed-jsdom this object was now available and jest was complaining that we have open listeners. That's why I added this line to be able to control when we need the broadcast listeners in the tests.